de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/ServerConfig.java

```diff
@@ -41,4 +42,16 @@ public static final String getPassword() {
     public static String getWorkspacePath() {
         return System.getProperty(WORKSPACE_PATH_KEY);
     }
+
+    /**
+     * Returns if the user has requested an interactive console.
+     *
+     * @return if the user has requested an interactive console
+     */
+    public static boolean getInteractive() {
```

Comment:
could this be named 'isInteractive', 'hasToBeInteractive' or similar to show this is a property?